naga/src/front/wgsl/error.rs

```diff
@@ -1394,6 +1395,14 @@ impl<'a> Error<'a> {
                     crate::valid::MAX_TYPE_SIZE
                 )],
             },
+            Error::MissingIncomingPayload(span) => ParseError {
+                message: "incoming payload is missing on ray hit or miss shader entry point".to_string(),
```

Comment:
Could we be specific and say `anyhit`, `closesthit` or `miss` shader entry point.